lib/zendesk_api/resources.rb

```diff
@@ -857,4 +857,14 @@ class Variant < ZendeskAPI::Resource
       has_many Variant
     end
   end
+
+  class PushNotificationDevice < DataResource
+    def self.destroy_many(client, tokens)
+      ZendeskAPI::Collection.new(
+        client, self,"push_notification_devices" => tokens,
+        :path =>  "push_notification_devices/destroy_many",
```

Comment:
There's some weird spacing on these two lines:
`self,"push..."` and `:path => __ "push..."`